.eslintrc

```diff
@@ -21,7 +21,7 @@
     "eqeqeq": [2, "smart"],
     "indent": [2, 2],
     "keyword-spacing": 2,
-    "linebreak-style": [2, "unix"],
+    "linebreak-style": [2, "windows"],
```

Comment:
@architajain12 what's the rationale for this change?

Reply:
@miketaylr  I was using a windows pc and `npm run eslint` gave me linebreak errors so I had to change it to "windows". I'm reverting it to "unix" right now and sending you another PR.